### PR TITLE
setup_pemserver

### PR DESCRIPTION
**Pull request by vincentp7212**
Testing framework setup_pemserver

### CODE REVIEW

File: tests/tests/test_setup_efm.py [hunk 1]
```diff
@@ -61,4 +62,34 @@ def test_setup_efm_pg_read_all_settings():
...
+    host = get_primary()
+    packages = [
+        'openjdk-8-jdk',
+        'bsd-mailx',
+        'edb-efm44'
+    ]
+
+    for package in packages:
+        assert host.package(package).is_installed, \
+            "Package %s not installed" % packages
```
Comment (jt-edb) [inline tests/tests/test_setup_efm.py:95]: Including changes not related to the main subject of the PR is not a good practice. Please open a new PR if you want to apply changes on EFM tests.

File: tests/cases/setup_pemserver/inventory.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+all:
+  children:
+    primary:
+      hosts:
+        primary1:
+          ansible_host: {{ inventory_vars['primary1_ip'] }}
+          private_ip: {{ inventory_vars['primary1_ip'] }}
```
Comment (jt-edb) [inline tests/cases/setup_pemserver/inventory.yml.j2:8]: I don't think we need a `primary` node if we are going to test PEM server only. Could you confirm?

**Review by jt-edb — CHANGES_REQUESTED** [review]
Please see my comments.

File: tests/cases/setup_pemserver/inventory.yml.j2 (hunk 2 above)
Comment (vincentp7212) [reply tests/cases/setup_pemserver/inventory.yml.j2]: Does not need primary node

**Pull request merged**